docs/resources/Channel.md

```diff
@@ -13,7 +13,7 @@ Guild channels represent an isolated set of users and messages within a Guild.
 | id | snowflake | the id of this channel (will be equal to the guild if it's the "general" channel) | Always|
 | guild\_id | snowflake | the id of the guild | Always |
 | name | string | the name of the channel (2-100 characters) | Always |
-| type | string | "text" or "voice" | Always |
+| type | integer | see [channel types](#DOCS_CHANNEL/channel-types) | Always |
```

Comment:
should just say `the type of the channel` to match the others.. can hyper link type